excercise9/invoicetransaction/Main.java

```diff
@@ -0,0 +1,64 @@
+package invoicetransaction;
+
+import java.sql.Connection;
+import java.util.List;
+import java.util.Scanner;
+import jdbcutility.JDBCUtility;
+
+public class Main {
+	public static void main(String[] args) {
+Scanner scanner = new Scanner(System.in);
+		
+		int inv_id;
+		int item_id;
+		float price;
+		int qty;
+		
+		Connection invoiceTransactionConnection = JDBCUtility.getConnection();
```

Comment:
Can you add input validation and error handlers?

Reply:
added error handleres and input validation